console/src/components/UIKit/atoms/Tabs/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+
+import {
+  StyledTab,
+  StyledTabList,
+  StyledTabListItem,
+  StyledTabContent,
+} from './Tabs';
+
+export const Tabs = props => {
+  const [currentActiveTabIndex, changeCurrentActiveTab] = useState(0);
+  const { tabsData } = props;
+
+  const currentTabContent =
+    tabsData &&
+    tabsData.filter((_, index) => {
+      return index === currentActiveTabIndex;
+    });
+
+  if (tabsData && tabsData.length > 0) {
+    return (
+      <StyledTab {...props}>
+        {/* Tab Navigation */}
+        <StyledTabList>
+          {tabsData &&
+            tabsData.map(({ title }, index) => (
+              <StyledTabListItem
+                key={title}
+                selected={index === currentActiveTabIndex && true}
```

Comment:
Why `&& true`?

Reply:
when index === curentActiveTabIndex then I'm passing true as a prop for the selected prop.

Reply:
or do I need to use it like index === currentActiveTabIndex ? true : false.

Reply:
But... `index === currentActiveTabIndex` already returns `true` or `false`.

Reply:
:pepe_ok: